README.md

```diff
@@ -1,17 +1,22 @@
-[![NPM][npm]][npm-url]
-[![Deps][deps]][deps-url]
-[![Tests][travis]][travis-url]
-[![Coverage][cover]][cover-url]
+[![npm][npm]][npm-url]
+[![node][node]][node-url]
```

Comment:
Maybe we not be include so many badges?

I think that would be enough:

  * npm
  * deps
  * tests
  * coverage
  * code style